### PR TITLE
[T-63] Complete User API

### PR DESCRIPTION
**Pull request by lakatop**

### CODE REVIEW

**Review by michalpulpan — APPROVED** [review]
Nice! You could've used similar structure for shipping/billing info views like @VL-CZ did in `CartUpdateInfoBaseStorefrontView`, but I guess that's not a 🥇 priority 😄

**Review by michalpulpan — APPROVED** [review]
Nice! You could've used similar structure for shipping/billing info views like @VL-CZ did in `CartUpdateInfoBaseStorefrontView`, but I guess that's not a 🥇 priority 😄

File: src/backend/core/cart/views.py [hunk 1]
```diff
@@ -117,6 +117,9 @@ class CartCreateStorefrontView(APIView):
     def post(self, request):
         cart = Cart.objects.create()
         # TODO: set user if logged-in
```
Comment (VL-CZ) [inline src/backend/core/cart/views.py:119]: we may remove this TODO now 🙂

File: src/backend/core/country/urls.py [hunk 2]
```diff
@@ -12,4 +12,8 @@
     path("<str:code>/", views.CountryDetailView.as_view()),
     path("address/shipping/<int:id>/", views.ShippingAddressDetailView.as_view()),
     path("address/billing/<int:id>/", views.ShippingAddressDetailView.as_view()),
+    path("address/shipping/", views.ShippingInfoListView.as_view()),
```
Comment (VL-CZ) [inline src/backend/core/country/urls.py:15]: I'd vote for having these URLs prefixed by either `dashboard` or `storefront` to distinguish their usage, like in other views.
I suppose that all of these views are intended for dashboard, so I'd prefix them with `dashboard`

File: src/backend/core/cart/views.py (hunk 1 above)
Comment (lakatop) [reply src/backend/core/cart/views.py]: Removed.

File: src/backend/core/country/urls.py (hunk 2 above)
Comment (lakatop) [reply src/backend/core/country/urls.py]: Fixed.

**Review by VL-CZ — APPROVED** [review]

**Pull request merged**